examples/erc1155/src/lib.rs

```diff
@@ -42,4 +44,38 @@ impl Erc1155Example {
         self.erc1155._mint_batch(to, token_ids, amounts, &data)?;
         Ok(())
     }
+
+    pub fn set_operator_approvals(
```

Comment:
I do not like having functions for testing in examples.

Reply:
This was part of the original branch, used for [e2e tests using invalid senders](https://github.com/OpenZeppelin/rust-contracts-stylus/pull/417/files#diff-12b1dcda468f3b063e8f05359d5bd72dd0be1248a15b559f6be91c6df018a798R1670), so I left it here.
Though on the second look, these tests seem redundant as the Burnable implementation prevents `Address::ZERO` being used as the sender account anyway (no one will ever be approved by the zero address to manage its funds). What do you think?

Reply:
I would remove this fn and use the contract in a proper way as in Solidity.